manifests/server/database.pp

```diff
@@ -2,14 +2,13 @@
 define postgresql::server::database(
   $comment          = undef,
   $dbname           = $title,
-  $owner            = $postgresql::server::user,
+  $owner            = undef,
   $tablespace       = undef,
   $template         = 'template0',
   $encoding         = $postgresql::server::encoding,
   $locale           = $postgresql::server::locale,
   $istemplate       = false,
   $connect_settings = $postgresql::server::default_connect_settings,
-  $change_ownership = false,
```

Comment:
For reference, this is a revert of https://github.com/hasegeli/puppetlabs-postgresql/commit/82c6716d4845bde01a03c58b2123b3009c0d97ca which indeed was never released.